sandbox/pc2syscheck.cmd

```diff
@@ -0,0 +1,25 @@
+@echo off
+REM Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+
+rem Purpose: start the team module
```

Comment:
I'm not at all sure what the purpose of this .cmd file IS, but I'm pretty sure it's NOT to "start the team module" as stated by the above line.  The file needs a comment which (correctly) indicates its purpose.

In particular, it also needs an explanation of why it is setting a whole bunch of variables and why it is returning FAIL_EXIT_CODE.

Reply:
This is the script (batch file) that is used to see if sandbox's are supported.  On windows, they are not.  But, the framework is there (with return codes) for when they are supported.  I have no idea where that comment came from about team module... I must have grabbed "pc2team.bat" as a starting point.  Of course, I'll have to fix that comment with a description of what the thing is **SUPPOSED** to do.

Reply:
Fixed.